README.md

```diff
@@ -1,4 +1,51 @@
 ## Config Service
 Service for storing and serving configurations across multiple services and contexts.
 
-Refer [config_service.proto](config-service-api/src/main/proto/org/hypertrace/config/service/v1/config_service.proto) for Config Service APIs
\ No newline at end of file
+## Description
+
+[config service](https://github.com/hypertrace/config-service) is a place to store configuration data such as user preferences, saved query filters, ingestion config, etc. Many of these use cases have not yet been built out. In general, this service is meant for user-managed configuration that needs to be persisted, and contains support for version history, auditing etc. In the past, we've addressed such things by spinning up individual services (such as attribute service). As new features get built out, we want to avoid that (and eventually to merge older services back into this).
+
+| ![space-1.jpg](https://hypertrace-docs.s3.amazonaws.com/arch/ht-arch.png) | 
+|:--:| 
+| *Hypertrace Architecture* |
+
+Refer [config_service.proto](config-service-api/src/main/proto/org/hypertrace/config/service/v1/config_service.proto) for Config Service APIs.
+
+## Building locally
+The Attribute service uses gradlew to compile/install/distribute. Gradle wrapper is already part of the source code. To build Attribute Service, run:
```

Comment:
Need to replace `Attribute service` by `Config Service` (at 2 places in the same line).

Reply:
Done!